app/models/preserved_object.rb

```diff
@@ -68,7 +67,7 @@ def audit_moab_version_replication!
   # Of those eligible for replication, use whichever was checksum validated most recently.
   # @return [String, nil] The storage location (storage_root/storage_trunk) where the Moab that should be replicated lives.
   def moab_replication_storage_location
-    moabs_eligible_for_replication.joins(:moab_storage_root).order(last_checksum_validation: :desc).limit(1).pluck(:storage_location).first
+    moabs_eligible_for_replication.joins(:moab_storage_root).order(last_checksum_validation: :desc).limit(1).pick(:storage_location)
```

Comment:
Won't this now return an Array, where it returned a scalar before?

Reply:
pick takes the first:  https://til.hashrocket.com/posts/wcwerkn2il-rails-6-new-activerecord-method-pick